.eslintrc.cjs

```diff
@@ -0,0 +1,194 @@
+		"no-dupe-else-if": 1, // Disallow duplicate conditions in if-else-if chains
+		"no-dupe-keys": 1, // Disallow duplicate keys in object literals
+		"no-duplicate-case": 1, // Disallow duplicate case labels
+		"no-empty-character-class": 1, // Disallow empty character classes in regular expressions
+		"no-empty-pattern": 1, // Disallow empty destructuring patterns
+		"no-ex-assign": 1, // Disallow reassigning exceptions in `catch` clauses
+		"no-extra-boolean-cast": 1, // Disallow unnecessary boolean casts
+		"no-fallthrough": 1, // Disallow fallthrough of `case` statements
+		"no-func-assign": 1, // Disallow reassigning `function` declarations
+		"no-global-assign": 1, // Disallow assignments to native objects or read-only global variables
+		"no-import-assign": 1, // Disallow assigning to imported bindings
+		"no-inner-declarations": 1, // Disallow variable or `function` declarations in nested blocks
+		"no-invalid-regexp": 1, // Disallow invalid regular expression strings in `RegExp` constructors
+		"no-irregular-whitespace": 1, // Disallow irregular whitespace
+		"no-misleading-character-class": 1, // Disallow characters which are made with multiple code points in character class syntax
+		"no-new-symbol": 1, // Disallow `new` operators with the `Symbol` object
+		"no-nonoctal-decimal-escape": 1, // Disallow `\8` and `\9` escape sequences in string literals
+		"no-obj-calls": 1, // Disallow calling global object properties as functions
+		"no-octal": 1, // Disallow octal literals
+		"no-prototype-builtins": 1, // Disallow calling some `Object.prototype` methods directly on objects
+		"no-regex-spaces": 1, // Disallow multiple spaces in regular expressions
+		"no-self-assign": 1, // Disallow assignments where both sides are exactly the same
+		"no-shadow-restricted-names": 1, // Disallow identifiers from shadowing restricted names
+		"no-this-before-super": 1, // Disallow `this`/`super` before calling `super()` in constructors
+		"no-undef": 1, // Disallow the use of undeclared variables unless mentioned in `/*global */` comments
+		"no-unexpected-multiline": 1, // Disallow confusing multiline expressions
+		"no-unreachable": 1, // Disallow unreachable code after `return`, `throw`, `continue`, and `break` statements
+		"no-unsafe-finally": 1, // Disallow control flow statements in `finally` blocks
+		"no-unsafe-negation": 1, // Disallow negating the left operand of relational operators
+		"no-unsafe-optional-chaining": 1, // Disallow use of optional chaining in contexts where the `undefined` value is not allowed
+		"no-unused-labels": 1, // Disallow unused labels
+		"no-useless-backreference": 1, // Disallow useless backreferences in regular expressions
+		"no-useless-call": 1, // Disallow unnecessary calls to `.call()` and `.apply()`
+		"no-useless-catch": 1, // Disallow unnecessary `catch` clauses
+		"no-useless-escape": 1, // Disallow unnecessary escape characters
+		"no-with": 1, // Disallow `with` statements
```

Comment:
That's a syntax error in ESM/strict mode anyways, so not sure we need it?

Reply:
True... Not sure it hurts anything? Happy to remove this if you like.

Reply:
🤷🏽‍♀️

Happy to leave it